lib/checker.js

```diff
@@ -18,13 +18,11 @@ const checks = requireDir('./checks');
 const checker = function checkAll(themePath, options) {
     options = options || {};
 
-    const passedVersion = _.get(options, 'checkVersion', 'v2');
+    const passedVersion = _.get(options, 'checkVersion', 'v3');
     let version = passedVersion;
 
     // TOOD: get rid of 'latest' concept because its unnecessary invention
-    if (passedVersion === 'latest') {
-        version = 'canary';
-    } else if (passedVersion === 'v3') {
+    if (passedVersion === 'latest' || passedVersion === 'v4') {
```

Comment:
Maybe it's time to get rid of the `latest` as suggested above because we are going to bump the major version of `gscan` anyway?